src/scenes/Contacts/components/Decription/Description.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+
+import styles from './Description.module.scss';
+
+const Description = () => {
+  const data = useStaticQuery(graphql`
```

Comment:
as far as I know, @mikhailnetkachev-halolab keeping queries in https://github.com/Halo-Lab/halo-lab/tree/development/src/hooks/queries